### ISSUE TITLE
Insufficient documentation regarding keymaps

### ISSUE THREAD

**Issue by goncalves-diogo** [nvim-treesitter/nvim-tree-docs#8]
Hi, I'm currently trying to get this to work on my workflow and i can't reach any breakthrough.

My current configuration file has:

```  
tree_docs = {
    enable = true,
    keymaps = {
        doc_node_at_cursor = "<leader>a"
        },
  },
```
I was trying to remap doc_node_at_cursor to <leader>a. However i can't seem to get this to work. I've also ran `:nmap` in order to see all the normal mappings and it is not mapped at all

**Comment by Mte90** [comment]
I agree with that I see that also the default hotkey is not mapped in vim.

**Comment by steelsojka** [comment]
Since this is an nvim-treesitter module you can find the documentation for
it in that repo. As far as them not working, that would be bug. We'll need
to figure out why they arent working.

**Comment by nathanielks** [comment]
Noting for future readers: I haven't been able to figure out _why_, but I _think_ this plugin can't map when something else exists at the beginning of the map. That's a poor way of explaining, so hopefully this is clearer: the default mapping is `gdd`. If you have any mapping that begins with `g`, the mapping won't apply. I think this is also why mapping to `<leader>` doesn't work, either. The way I figured it out was by trying different mappings and found one that didn't have anything (for me, this was `fb`).
